bookstore/src/App.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable jsx-quotes */
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Category, Books, NotFound } from './components/component';
+
+const App = () => (
+  <div>
+    <Router>
+      <Routes>
+        <Route path='/books' index element={<Books />} />
+        <Route path='/category' element={<Category />} />
+        <Route path='*' element={<NotFound />} />
+      </Routes>
```

Comment:
- [ ] Good job so far 👍 Kindly remove the third route path since the requirements state that your app should have only 2 routes and the default route should be the `books` element.